extension/src/helpers/hooks/use-get-history.tsx

```diff
@@ -0,0 +1,85 @@
+import { useReducer } from "react";
```

Comment:
NIT: can you call this file useGetHistory.tsx to conform to the naming convention in the rest of the repo?